_includes/block/head.html

```diff
@@ -23,12 +23,29 @@
     <link rel="canonical" href="{{ site.url }}{{ page.url}}">
   {% endif %}
 
-  <meta name="og:image" content="{{ site.url }}/assets/images/banner.png">
+  <meta property="og:locale" content="en_US">
+  <meta property="og:site_name" content="{{ site.title }}">
+  <meta property="og:url" content="{{ site.url }}{{ page.url}}">
+
+  {% if page.cover %}
+    <meta name="og:image" content="{{ site.url }}{{ page.cover }}">
+  {% else %}
+    <meta name="og:image" content="{{ site.url }}/assets/images/banner.png">
+  {% endif %}
+
   {% if page.title %}
     <meta property="og:title" content="{% if page.title %}{{ page.title }} — {% endif %}{{ site.title }}">
```

Comment:
_nit:_ Since you are already in this file, would you mind fixing my redundant check around `page.title`? No problem if you don't. This line can just be:

```html
<meta property="og:title" content="{{ page.title }} — {{ site.title }}">
```